data/input_2022/Architecture/Results/node-wot.csv

```diff
@@ -0,0 +1,69 @@
+"arch-discovery-no-metadata-in-introduction-urls","not-impl", "User decision, URL based on interaction name"
+"arch-discovery-should-use-standard","not-impl",
+"arch-event-dataschema","null", "Do not understand the assertion"
+"arch-event-trigger","impl",
+"arch-form-iris","impl",
+"arch-form-iris2","impl",
+"arch-hypermedia","impl",
+"arch-hypermedia-caching","not-impl",
+"arch-hypermedia-protocol-binding","impl",
+"arch-intermediary-td-extra-protocols","null",
+"arch-media-type","impl",
+"arch-media-type-extra","impl",
+"arch-networked","impl",
+"arch-op-expected-request","impl",
+"arch-op-form-fields-protocol","impl",
+"arch-op-request-method","impl",
+"arch-other-thing-representations","not-impl", "May be done on top of it"
+"arch-privacy-consideration-access-control-mandatory-person","not-impl", "Use-case dependent"
+"arch-privacy-consideration-dist-td-auth","not-impl",
+"arch-privacy-consideration-explicit-pii","not-impl",
+"arch-privacy-consideration-id-access-control-mandatory-immutable","not-impl",
+"arch-privacy-consideration-min-explicit-pii","not-impl",
+"arch-privacy-consideration-no-unness-info-td","not-impl",
+"arch-property-dataschema","null", "Do not understand the assertion"
+"arch-rel-type-lowercase","null",
+"arch-rel-types","null",
+"arch-schema","null",
+"arch-security-consideration-auth-private-data","null",
+"arch-security-consideration-avoid-direct","impl",
+"arch-security-consideration-communication-binding","null",
+"arch-security-consideration-communication-platform","not-impl",
+"arch-security-consideration-dtls-1-2","null",
+"arch-security-consideration-dtls-1-3","null",
+"arch-security-consideration-hal-refuse-unsafe","null",
+"arch-security-consideration-isolation-sensitive","null",
+"arch-security-consideration-isolation-tenants","null",
+"arch-security-consideration-limit-cred-access","null",
+"arch-security-consideration-limit-trust","null",
+"arch-security-consideration-no-earlier-tls-or-dtls","null",
+"arch-security-consideration-no-expose-cred","null",
+"arch-security-consideration-no-private-security-data","null",
+"arch-security-consideration-other-programming-mechanisms","null",
+"arch-security-consideration-public-metadata-only","null",
+"arch-security-consideration-secure-cred-isolation","null",
+"arch-security-consideration-secure-cred-storage","null",
+"arch-security-consideration-secure-update","null",
+"arch-security-consideration-segmented-network","null",
+"arch-security-consideration-separate-security-data","null",
+"arch-security-consideration-tls-1-2","null",
+"arch-security-consideration-tls-1-3","null",
+"arch-security-consideration-tls-mandatory-pub","null",
+"arch-security-consideration-tls-optional-on-lan","null",
+"arch-security-consideration-tls-recommended-priv","null",
+"arch-security-consideration-use-hal","null",
+"arch-security-consideration-use-psk","null",
+"arch-td-linking","impl",
+"arch-td-mandatory","pass", "Do not understand the assertion"
```

Comment:
A WoT Thing should always offer a TD?

Reply:
This is simply the definition of a WoT Thing: an IoT device or service with a TD.   Anyway, I am going to merge and update the WoT Architecture Implementation Report.  However, feel free to make further updates.